src/sourmash/minhash.py

```diff
@@ -845,13 +845,17 @@ def similarity(self, other, ignore_abundance=False, downsample=False):
             downsample,
         )
 
-    def angular_similarity(self, other):
+    def angular_similarity(self, other, downsample=False):
         "Calculate the angular similarity."
         if not (self.track_abundance and other.track_abundance):
             raise TypeError(
                 "Error: Angular (cosine) similarity requires both sketches to track hash abundance."
             )
-        return self._methodcall(lib.kmerminhash_angular_similarity, other._get_objptr())
+        # return self._methodcall(lib.kmerminhash_angular_similarity, other._get_objptr())
```

Comment:
we should create an issue to remove (b/c unused) or fix (and then use) `lib.kmerminhash_angular_similarity`